selfdrive/car/honda/interface.py

```diff
@@ -75,6 +75,7 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
       ret.longitudinalActuatorDelayUpperBound = 0.5 # s
       if candidate in HONDA_BOSCH_RADARLESS:
         ret.stopAccel = CarControllerParams.BOSCH_ACCEL_MIN  # stock uses -4.0 m/s^2 once stopped but limited by safety model
+        ret.stoppingDecelRate = 0.15  # reach stopping target smoothly
```

Comment:
this could be too slow for hills, I would pick a value that largely improves thing -- it doesn't need to be perfect. then we can more easily merge without testing extremes

Reply:
Sounds good, back to 0.3 for now, definitely an improvement without being perfect.